packages/js/product-editor/src/blocks/pricing/edit.tsx

```diff
@@ -38,51 +42,44 @@ export function Edit( { attributes }: { attributes: BlockAttributes } ) {
 	const { getCurrencyConfig, formatAmount } = context;
 	const currencyConfig = getCurrencyConfig();
 	const inputProps = useCurrencyInputProps( {
-		value: regularPrice,
-		setValue: setRegularPrice,
+		value: price,
+		setValue: setPrice,
 	} );
 
-	const taxSettingsElement = showPricingSection
-		? createInterpolateElement(
-				__(
-					'Manage more settings in <link>Pricing.</link>',
-					'woocommerce'
+	const interpolatedHelp = help
+		? createInterpolateElement( help, {
+				PricingTab: (
+					<Link
+						href={ getNewPath( { tab: 'pricing' } ) }
+						onClick={ () => {
+							recordEvent(
+								'product_pricing_list_price_help_tax_settings_click'
```

Comment:
Nice, thanks for adding tracks. Although not sure if this is related to the `help_tax_settings`, is that a copy/paste error maybe?
We may want to simplify it a bit to: `product_pricing_help_click`?

Reply:
Well that was not part of the scope of this PR but it was solved here anyway -> https://github.com/woocommerce/woocommerce/pull/37961/commits/db3bb644b2038bb1b9d89146c5a67316906e34b6